var/spack/repos/builtin/packages/py-jedi/package.py

```diff
@@ -36,18 +37,11 @@ class PyJedi(PythonPackage):
     )
     version("0.9.0", sha256="3b4c19fba31bdead9ab7350fb9fa7c914c59b0a807dcdd5c00a05feb85491d31")
 
-    depends_on("py-setuptools", type=("build", "run"))
+    depends_on("py-setuptools", type="build")
```

Comment:
Setuptools may actually be used at run-time. I see pkg_resources imported in a few places. Unclear if they're only for testing or not.

Reply:
I have built all `py-jedi` versions with `--test=root` without problems, that's why I thought it would be ok to remove it. Should I still add it?

Reply:
I would add it. It's imported in a function, so it isn't tested with our import tests.